pkg/configuration/configuration.go

```diff
@@ -131,30 +137,42 @@ const (
 	DefaultVerificationMessageTemplate = "Your verification code for Red Hat Developer Sandbox is: %s"
 )
 
-// Registry encapsulates the Viper configuration registry which stores the
+// Config encapsulates the Viper configuration registry which stores the
 // configuration data in-memory.
-type Registry struct {
-	v *viper.Viper
+type Config struct {
+	v            *viper.Viper
+	secretValues map[string]string
 }
 
 // CreateEmptyRegistry creates an initial, empty registry.
-func CreateEmptyRegistry() *Registry {
-	c := Registry{
-		v: viper.New(),
+func CreateEmptyRegistry(cl client.Client) (*Config, error) {
+	os.Setenv("HOST_OPERATOR_SECRET_NAME", "host-operator-secret")
+	secret, err := configuration.LoadFromSecret("HOST_OPERATOR_SECRET_NAME", cl)
```

Comment:
Just to confirm. `LoadFromSecret()` won't return an error if there is no such secret, right?

Reply:
yeah, that's right, in this case since we are hard-coding the name we'll end up getting the `not found` error and returning an empty map with error set to nil: https://github.com/codeready-toolchain/toolchain-common/blob/master/pkg/configuration/load_configuration.go#L42-L45